### PR TITLE
Update gear menus and info pane headers.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

**Comment by adamjspooner** [comment]
Just kidding. This isn't ready for review.

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for review now.

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

**Comment by adamjspooner** [comment]
**Bug-like things**
* Fixed
* This is a bug currently in master (as far as I can tell). Let's TRIAGE it.
* This is a bug currently in master (as far as I can tell). Let's TRIAGE it.
* This is about to change. So, I'm not sure it's worth putting time into.

**Questions**
* We decided to remove red from most things, especially if there's a confirmation step (or they're not really that dangerous). There's a confirmation step when leaving a team.
* Fixed

File: shared/router-v2/account-switcher/index.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ export default Kb.HeaderHoc(AccountSwitcher)
 const styles = Styles.styleSheetCreate(() => ({
   buttonBox: Styles.padding(0, Styles.globalMargins.small, Styles.globalMargins.tiny),
   desktopScrollview: {
-    width: 200,
+    width: '100%',
```
Comment (thebearjew) [inline shared/router-v2/account-switcher/index.tsx]: Just curious. What that issue that needed full-width?

File: shared/chat/conversation/info-panel/menu/index.tsx [hunk 2]
```diff
@@ -124,37 +160,77 @@ class InfoPanelMenu extends React.Component<Props> {
...
+      items.push(channelItem, {
+        icon: 'iconfont-people',
+        onClick: props.onViewTeam,
+        title: 'Team info',
+      })
+      if (props.canAddPeople) {
+        addPeopleItems.forEach(item => items.push(item))
+      }
+      items.push({icon: 'iconfont-leave', onClick: props.onLeaveTeam, title: 'Leave team'})
+    }
```
Comment (thebearjew) [inline shared/chat/conversation/info-panel/menu/index.tsx]: Lose the `else` to reduce the nesting

```ts
if (isAdhoc) {
    ...
    items.push({
        danger: true,
        icon: 'iconfont-block-user',
        onClick: props.onBlockConv,
        title: 'Block',
    })
    return
}

if (!props.isSmallTeam && !props.hasHeader) {
    // ...
}

// ...
```

File: shared/router-v2/account-switcher/index.tsx (hunk 1 above)
Comment (adamjspooner) [reply shared/router-v2/account-switcher/index.tsx]: Floating menus increased to 240px-wide.

File: shared/chat/conversation/info-panel/menu/index.tsx (hunk 2 above)
Comment (adamjspooner) [reply shared/chat/conversation/info-panel/menu/index.tsx]: I think that would be useful if moving it to a separate function would clean things up, but that function would either need to have all the items preceding it passed in or created within it. So, either a messy function call or a the same mess further up/down the page. Unless I'm misunderstanding your feedback.

**Pull request merged**